index.d.ts

```diff
@@ -28,6 +32,7 @@ declare module 'jwks-rsa' {
       cacheMaxAge?: number;
       jwksRequestsPerMinute?: number;
       strictSsl?: boolean;
+      headers?: Headers;
```

Comment:
can we name this `requestHeaders` instead?

Reply:
Done 🙂